packages/python-argus/PKGBUILD

```
@@ -0,0 +1,48 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+# See COPYING for license details.
+
+pkgname=python-argus
+_pkgname=Argus
+__pkgname=argus
+pkgver=7.56ece87
+pkgrel=1
+pkgdesc='All-in-One OSINT Toolkit.'
+arch=('any')
+groups=('blackarch' 'blackarch-misc' 'blackarch-recon')
+url='https://github.com/jasonxtn/Argus'
+license=('GPL')
+depends=('python')
+makedepends=('git' 'python-setuptools' 'python-pip')
```

```suggestion
makedepends=('git')
```

and do not use a virtual environment please